extensions/cornerstone/src/commandsModule.ts

```diff
@@ -72,7 +96,129 @@ const commandsModule = ({ servicesManager }) => {
     return toolGroup;
   }
 
+  function _showViewerContextMenu(viewerElement, options) {
+    let defaultPointsPosition = [];
+    if (options.nearbyToolData) {
+      defaultPointsPosition = commandsManager.runCommand(
+        'getToolDataActiveCanvasPoints',
+        { toolData: options.nearbyToolData }
+      );
+    }
+
+    contextMenuController.showContextMenu(
+      options,
+      viewerElement,
```

Comment:
what is viewerElement, is it enabledElement in cs3d language? there is no viewer element. You can also say viewportElement